casanovo/denovo/model.py

```diff
@@ -989,6 +989,190 @@ def configure_optimizers(
         return [optimizer], {"scheduler": lr_scheduler, "interval": "step"}
 
 
+class DBSpec2Pep(Spec2Pep):
+    """
+    Inherits Spec2Pep
+
+    Hijacks teacher-forcing implemented in Spec2Pep and
+    uses it to predict scores between a spectra and associated peptide.
+    Input format is .mgf, with comma-separated targets
+    and decoys in the SEQ field. Decoys should have a prefix of "decoy_".
```

Comment:
Discussion on 6/18/24 MS Meeting: Decided that support for custom decoy tags would not be needed at the moment (or ever, since we don't plan to do FDR control within this tool.)